assets/js/modules/Geocoding.js

```diff
@@ -0,0 +1,53 @@
+export default class Geocoding {
```

Comment:
Performance Tipp: Verwende keine JavaScript Klassen mit Babel:
https://css-tricks.com/avoid-heavy-babel-transformations-by-sometimes-not-writing-modern-javascript/

Reply:
Uff, danke für den Tipp. Das war mir so nicht bewusst :-(